__tests__/app.test.js

```diff
@@ -66,18 +66,67 @@ describe("GET /api/reviews/:review_id", () => {
     return request(app)
       .get("/api/reviews/grace")
       .expect(400)
-      .then((response) => {
-        const { message } = response.body;
-        expect(message).toBe("Bad request: Invalid data type.");
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad request: Invalid data type.");
       });
   });
   test("404: responds with does not exist message when the review_id doesn't exist", () => {
     return request(app)
       .get("/api/reviews/99999")
       .expect(404)
-      .then((response) => {
-        const { message } = response.body;
-        expect(message).toBe("The review_id does not exist.");
+      .then(({ body }) => {
+        //const { message } = response.body;
+        expect(body.message).toBe("The review_id does not exist.");
+      });
+  });
+});
+
+describe("PATCH /api/reviews/:review_id", () => {
+  test("201: responds with the correct updated object", () => {
+    return request(app)
+      .patch(`/api/reviews/3`)
+      .send({ inc_votes: 5 })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.review).toEqual({
+          review_id: 3,
+          title: "Ultimate Werewolf",
+          designer: "Akihisa Okui",
+          owner: "bainesface",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          review_body: "We couldn't find the werewolf!",
+          category: "social deduction",
+          created_at: "2021-01-18T10:01:41.251Z",
+          votes: 10,
+        });
+      });
+  });
+  test("404: responds with does not exist message when the review_id doesn't exist", () => {
+    return request(app)
+      .patch(`/api/reviews/99999`)
+      .send({ inc_votes: 5 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("The review_id does not exist.");
+      });
+  });
+  test("400: responds with a bad request message when the review_id is given as an invalid data type", () => {
+    return request(app)
+      .patch("/api/reviews/grace")
+      .send({ inc_votes: 5 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad request: Invalid data type.");
+      });
+  });
+  test("400: responds with a bad request message when the inc_votes is given as an invalid data type", () => {
+    return request(app)
+      .patch("/api/reviews/3")
+      .send({ inc_votes: "five" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad request: Invalid data type.");
       });
   });
 });
```

Comment:
Hey! One more test here I would say. What would happen if a user tried to send across a body that didn't contain a key of `inc_votes`?